grails-app/controllers/au/org/ala/LoginInterceptor.groovy

```diff
@@ -0,0 +1,117 @@
+package au.org.ala
```

Comment:
ALA copyright message should be here

Reply:
Package is incomplete - normally needs to have the application name as part of package - e.g. `au.org.ala.spatial.service`. See controllers or services in this app for reference.

Reply:
Unless the `ala-ws-security-plugin:2.0` used this package name...